tests/BuildScriptGenerator.Tests/Node/NodeBuildScriptGenerationTest.cs

```diff
@@ -452,14 +453,161 @@ public void GeneratedScript_DoesNotZipNodeModules_IfZipNodeModulesEnvironmentVar
             Assert.True(scriptGenerator.IsCleanRepo(repo));
         }
 
+        [Theory]
+        [InlineData("4.4.7")]
```

Comment:
move this list of node versions to the GenerateTestVAlues class, so we don't have to duplicate its values and we document what they mean